var/spack/repos/builtin/packages/py-packaging/package.py

```diff
@@ -5,20 +5,34 @@
 
 from spack import *
 
-
 class PyPackaging(PythonPackage):
```

Comment:
Can you move the changes to `py-packaging` to a separate PR? I'm not sure how useful these old versions are anyway...

Reply:
I've reverted my change to py-packaging and removed py-statsmodels@0.13.2 which requires py-packaging@21.3